src/test/java/core/basesyntax/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,175 @@
+
+    @BeforeAll
+    public static void beforeAll() {
+        registrationService = new RegistrationServiceImpl();
+    }
+
+    @BeforeEach
+    public void setUp() {
+        storageDao = new StorageDaoImpl();
+    }
+
+    @Test
+    public void addUserToList_Ok() {
+        storageDao.add(FIRST_USER);
+        storageDao.add(SECOND_USER);
+        storageDao.add(THIRD_USER);
+
+        User actualFirstUser = storageDao.get(FIRST_USER.getLogin());
+        User actualSecondUser = storageDao.get(SECOND_USER.getLogin());
+        User actualThirdUser = storageDao.get(THIRD_USER.getLogin());
+
+        assertEquals(FIRST_USER, actualFirstUser,
+                "Test failed! First user should be " + FIRST_USER);
+        assertEquals(SECOND_USER, actualSecondUser,
+                "Test failed! Second user should be " + SECOND_USER);
+        assertEquals(THIRD_USER, actualThirdUser,
+                "Test failed! Third user should be " + THIRD_USER);
+    }
+
+    @Test
+    public void testRegisterAddUserIsNull_NotOk() {
+        assertThrows(RegistrationException.class, () -> {
+            registrationService.register(null);
+
+        });
+    }
+
+    @Test
+    public void testRegisterAddUserWithSameLogin_NotOk() {
+        assertThrows(RegistrationException.class, () -> {
+            registrationService.register(SAME_LOGIN_WITH_FIRST_USER);
+        });
+    }
+
+    @Test
+    public void testRegisterAddUserWithLoginOrPasswordLengthLessThanSix_NotOk() {
+        assertThrows(RegistrationException.class, () -> {
+            registrationService.register(USER_WITH_LOGIN_LESS_THAN_SIX);
+            registrationService.register(USER_WITH_PASSWORD_LESS_THAN_SIX);
+        });
+    }
+
+    @Test
+    public void testRegisterAddUserWithAgeLessThanMinAge_NotOk() {
+        assertThrows(RegistrationException.class, () -> {
+            registrationService.register(USER_WITH_AGE_LESS_THAN_MIN_AGE);
+        });
+    }
+
+    @Test
+    public void testAddUserThanOneOfTheParametersNull_NotOk() {
+        User userExpectedFirst = storageDao.add(USER_WITH_AGE_NULL);
+        User userExpectedSecond = storageDao.add(USER_WITH_LOGIN_NULL);
+        User userExpectedThird = storageDao.add(USER_WITH_PASSWORD_NULL);
+
+        User actualFirstResultNull = storageDao.get(USER_WITH_AGE_NULL.getLogin());
+        User actualSecondResultNull = storageDao.get(USER_WITH_LOGIN_NULL.getLogin());
+        User actualThirdResultNull = storageDao.get(USER_WITH_PASSWORD_NULL.getLogin());
+
+        assertNotEquals(userExpectedFirst, actualFirstResultNull,
+                "Test failed! Age cannot be null");
+        assertNotEquals(userExpectedSecond, actualSecondResultNull,
+                "Test failed! Login cannot be null");
+        assertNotEquals(userExpectedThird, actualThirdResultNull,
+                "Test failed! Password cannot be null");
+    }
+
+    @Test
+    public void testAddUserWithAgeLessThanMinAge_NotOk() {
```

Comment:
```suggestion
    public void testAddUserWithAgeLessThanMinAge_notOk() {
```
use lowercase after underscore everywhere